src/codeu/chat/util/PasswordStorage.java

```diff
@@ -0,0 +1,241 @@
+package codeu.chat.util;
+import codeu.chat.client.core.Context;
+
+import java.io.Console;
+import java.security.SecureRandom;
+import javax.crypto.spec.PBEKeySpec;
+import javax.crypto.SecretKeyFactory;
+import java.math.BigInteger;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Arrays;
+import javax.xml.bind.DatatypeConverter;
+
+/***
+ * Password encryption class from https://github.com/defuse/password-hashing
+ */
+
+public class PasswordStorage
```

Comment:
PasswordStorage indicates that this class stores passwords, but it's not...make the name more descriptive: PasswordUtils?